seeder-start-restart.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Start-restart script for Kiwix-seeder
+#
+# Assumes an optional (yet recommended) config file in /etc/seeder.config
+# to overwrite the following variables.
+
+CONTAINER_NAME="seeder"                             # name of docker container
+IMAGE="ghcr.io/kiwix/bittorrent-seeder:latest"      # docker image to use
+
+DATA_PATH=$(pwd)/kiwix-seeder                       # path to store ZIM files (and incomplete .!qB ones) in (there's no hierarchy)
+MAX_STORAGE="10GiB"									# maximum disk-space to use
+SLEEP_INTERVAL="5m"                                 # how long to pause in-between catalog checks when using --loop
+DEBUG=""                                            # whether to print debug logs (set to 1 to enable)
+SEED_WHOLE_CATALOG=									# whether to continue if filters (or lack of) end up seeding the whole catalog (prevents accidental no-filter launch)
+
+#
+# FILTERS
+# - use glob-patterns
+# - split using pipe (|)
+# - combined with same filter
+FILENAMES=""										# matching filename (filepath actually)
+LANGUAGES=""										# matching language (ISO-639-3 language codes)
+CATEGORIES=""										# matching Category
+FLAVOURS=""											# matching Flavour metadata (nodet, mini, nopic, maxi)
+TAGS=""												# containing following tag(s)
+AUTHORS=""											# matching Creator metadata
+PUBLISHERS=""										# matching Publisher metadata
+MIN_SIZE=""											# only ZIM larger than
+MAX_SIZE=""											# only ZIM smaller than
+
+# the following applies to those using the in-container qBittorrent
+# If using a remote qBittorrent instance, see NO_DEAMON below
+QBT_TORRENTING_PORT=6901                            # port to use for bittorrent. **MUST** be manually opened and forwarded to this host's IP as uPNP would not work accross docker
+QBT_PASSWORD=""                                     # qBittorrent WebUI password. If empty, one will be gen and printed
+WEBUI_PORT=8000                                     # port on this host to map to the qBittorrent WebUI (so you can monitor it)
```

Comment:
Default should probably be `127.0.0.1:8000` so that we expose WEBUI only on localhost, available only for kiwix-seeder operation by default.

Reply:
As explained above, #2 will allow this but I dont think it should be the default. WebUI is extremely useful to oversee what's happening and binding to localhost would prevent it from being accessed by the user.